core/router.js

```diff
@@ -117,7 +118,7 @@ module.exports = function (logger) {
 
         // there is no available transport or handler for this mu id, this should never happen, discard the packet...
         logger.error(message, 'routing error no available response transport function for')
-        cb('routing error no available response transport function for: ' + JSON.stringify(message))
+        cb('routing error no available response transport function for: ' + stringify(message))
```

Comment:
I'm 👎 on this change.

We should really return an `Error` as first object, and follows the node conventions. The user can still return an `Error`, and so our code needs to handle all of this properly.